src/scanoss/inspection/undeclared_component.py

```
@@ -1,38 +1,49 @@
 from .policy_check import PolicyCheck, PolicyStatus
 
+
 class UndeclaredComponent(PolicyCheck):
     """
     SCANOSS UndeclaredComponent class
     Inspects for undeclared components
     """
 
-    def __init__(self, debug: bool = False, trace: bool = True, quiet: bool = False, filepath: str = None,
-                 format_type: str = 'json', status: str = None, output: str = None, sbom_format: str = 'settings'):
+    def __init__(
```

Ruff (0.8.2)

Too many arguments in function definition (8 > 5)

(PLR0913)